<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat with Ai</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="chatDisplay">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userInputEntry"/>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fileModeButton">
      <property name="text">
       <string>Enable File Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="changeFolderButton">
      <property name="text">
       <string>Change Local Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="webSearchButton">
      <property name="text">
       <string>Enable Web Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>